app/facades/nav_facade.rb

```diff
@@ -14,7 +14,7 @@ def self.get_directions(start_address, end_address)
   end
 
   def self.trip_details(start_address, end_address)
-    details = route_directions(start_address, end_address)[:route].slice(:realTime, :fuelUsed, :hasTollRoad, :distance)
+    details = route_directions(start_address, end_address)[:route].slice(:realTime, :hasTollRoad, :distance)
```

Comment:
It's still wild to me that they removed fuelUsed in the time we were looking at it haha